dog.js

```diff
@@ -0,0 +1,11 @@
+function showInfo(name, breed, cost) {
+  alert(name + ', ' + breed + ', '+ 'is cost '+ cost);
+}
+
+
+let total = 0;
+
+function addTotal(cost) {
+  total += cost;
```

Comment:
You need to make sure you are treating total as a Number, so Math works correctly.
This is called sanitizing your inputs:

`total += Number(cost);`

Reply:
Got it. Changed in PC doc and will update later.